roles/openshift_logging/tasks/oc_apply.yaml

```diff
@@ -49,4 +49,4 @@
     failed_when: "'error' in generation_apply.stderr"
     changed_when: generation_apply.rc == 0
     when: "'field is immutable' in generation_apply.stderr"
-  when: file_content.kind == "Service"
+  when: file_content.kind in ["Service", "Route"]
```

Comment:
its to address a bug I came across while doing some quick local tests... changing the kibana hostname after deploying initially ran into the same issue we see with services